src/mongocxx/test/v1/exception.hh

```diff
@@ -0,0 +1,34 @@
+// Copyright 2009-present MongoDB, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <mongocxx/v1/exception.hpp>
+
+//
+
+#include <catch2/catch_tostring.hpp>
+
+CATCH_REGISTER_ENUM(
+    mongocxx::v1::source_errc,
+    mongocxx::v1::source_errc::zero,
+    mongocxx::v1::source_errc::mongocxx,
+    mongocxx::v1::source_errc::mongoc,
+    mongocxx::v1::source_errc::mongocrypt)
```

Comment:
```suggestion
    mongocxx::v1::source_errc::mongocrypt,
    mongocxx::v1::source_errc::server)
```

Reply:
Fixed in [`4ae0cd9` (#1469)](https://github.com/mongodb/mongo-cxx-driver/pull/1469/commits/4ae0cd904b70bff0154b791e68bd43db1a0dbbe5).